### ISSUE TITLE
label for Dropdown component is not read by screen reader

### ISSUE THREAD

**Issue by mattmazzola** [OfficeDev/office-ui-fabric-react#5347]
### Bug Report 
- __Package version(s)__: 
"office-ui-fabric-react": "5.100.0",

- __Browser and OS versions__:
Chrome Version 67.0.3396.87 (Official Build) (64-bit)
NonVisual Desktop Access (NVDA) Version: 2018.2.1

#### Priorities and help requested:
 
**Are you willing to submit a PR to fix?** No
 
**Requested priority:** Low
 
**Products/sites affected:** (if applicable)
Conversation Learner
 
#### Describe the issue:

We recently want through accessibility testing and a bug was logged that when Dropdown component is focused the label is not read by the screen reader.

See the video:
https://1drv.ms/v/s!AlIBzruyxcE4nIdt5K0WwYap4x69nw

Notice in example 1 which does not have ariaLabel attribute provided
I select text and then press tab to focus the dropdown. NVDA reads "list collapsed <placeholder>"

In example 2, which does have ariaLabel it reads "<label> list collapsed <placeholder>"
 
#### Actual behavior:
Label is not associated with component and is not read by screen reader
 
#### Expected behavior:
Label should be read by screen reader.  Perhaps when label is provided it should also assign to the ariaLabel.  This would cover the scenario and still allow flexibility.
 
### If applicable, please provide a codepen repro:
https://codepen.io/mattmazzola/pen/qKMzwE

**Comment by cliffkoh** [comment]
This is fixed in #5385. Please do feel free to review the PR or the animated screen capture and let me know if the captured behavior doesn't meet your expectation.

**Comment by dzearing** [comment]
Closing as it has been resolved. Please feel free to re-open if needed.
